Artesian/_Services/MarketDataService.py

```diff
@@ -6,19 +6,23 @@
 import itertools
 
 class MarketDataService:
-    """ A MarketData Entity represents a data curve enriched by its metadata.
-    each entity is composed of some fundamental parameters:
+    """ 
+        A MarketData Entity represents a data curve enriched by its metadata.
+        Each entity is composed of some fundamental parameters:
+
+        Attributes:
+            artesianConfig: The Artesian Configuration.
```

Comment:
this class doesn't have this public Attribute.
The arg of the __init__ is stored in a private __config thus not an Attribute.